dandiapi/api/management/commands/migrate_schema_versions.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import logging
+
+from dandischema.consts import DANDI_SCHEMA_VERSION
+from django.db import transaction
+import djclick as click
+
+from dandiapi.api.models import Version
+from dandiapi.api.services import audit
+from dandiapi.api.services.metadata import validate_version_metadata
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument(
+    'dandisets',
+    type=click.INT,
+    nargs=-1,
+)
+@click.option(
+    '-a',
+    '--all',
+    'include_all',
+    is_flag=True,
+    help='Run on all dandisets.',
+)
+def migrate_schema_versions(dandisets: tuple[int, ...], *, include_all: bool):
+    if bool(dandisets) == include_all:
+        raise click.ClickException("Must specify exactly one of 'dandisets' or --all")
+
+    versions = Version.objects.filter(version='draft')
+    if dandisets:
+        versions = versions.filter(dandiset_id__in=dandisets)
+
+    logger.info('Running migration for %s versions', versions.count())
+
+    for version in versions.iterator():
+        logger.info('-----------------------------------------')
+        with transaction.atomic():
+            locked_version = Version.objects.select_for_update().get(id=version.id)
+            locked_version.metadata['schemaVersion'] = DANDI_SCHEMA_VERSION
+            locked_version.status = Version.Status.PENDING
+            locked_version.save()
+
+            audit.update_metadata(
+                dandiset=locked_version.dandiset,
+                metadata=locked_version.metadata,
+                user=None,
+                admin=True,
+                description=f'Update schema version to {DANDI_SCHEMA_VERSION}',
+            )
```

Comment:
@waxlamp I opted to make use of the `admin` feature of our audit record system. Does the inclusion of an audit record at all make sense here? I would think so, since the underlying metadata is still changing, but I'd like a second opinion.